mozilla/blurts-server
# Add layout for authenticated pages

# Description

Adds the layout for pages that we use for authenticated pages.

# Screenshot

<img width="1400" alt="pr-authenticated-layout" src="https://github.com/mozilla/blurts-server/assets/13835474/5969df75-e717-4c79-bad4-f1e2866e5339">


# How to test

- Auth with FxA visiting `/oauth/init`
- Visit `/user/breaches` and make sure the user menu shows the current users data

# Checklist (Definition of Done)
